Get collection exercises by survey ids

arroyoAle opened the pull request:
# What and why?

# How to test?

# Jira

anwilkie commented:
/deploy wilkia

arroyoAle commented:
> So I don't believe this will work if there are no enrolments and will throw an exception. I also am not sure why this is still stored in redis, hasn't that been made redundant? I am also wonder what exercise /surveys is also being used for (if any) as it has been extended

Updated to be able to handle no enrolments and removed redis. I think you mean the old endpoint in collection exercise? In which case its still used by rOps, party and the acceptance tests

LJBabbage commented:
> > So I don't believe this will work if there are no enrolments and will throw an exception. I also am not sure why this is still stored in redis, hasn't that been made redundant? I am also wonder what exercise /surveys is also being used for (if any) as it has been extended
> 
> Updated to be able to handle no enrolments and removed redis. I think you mean the old endpoint in collection exercise? In which case its still used by rOps, party and the acceptance tests

It probably should be on the other PR. You added case events to be returned from 
```@RequestMapping(value = "/surveys", method = RequestMethod.GET, produces = "application/json")``` 
 I wondered what other things that is used for and if there is a performance hit

arroyoAle commented:
> > > So I don't believe this will work if there are no enrolments and will throw an exception. I also am not sure why this is still stored in redis, hasn't that been made redundant? I am also wonder what exercise /surveys is also being used for (if any) as it has been extended
> > 
> > 
> > Updated to be able to handle no enrolments and removed redis. I think you mean the old endpoint in collection exercise? In which case its still used by rOps, party and the acceptance tests
> 
> It probably should be on the other PR. You added case events to be returned from `@RequestMapping(value = "/surveys", method = RequestMethod.GET, produces = "application/json")` I wondered what other things that is used for and if there is a performance hit

Oh I see, I've also updated the other pr so if no survey ids are passed it returns a 204, and the `/surveys` endpoint we are now using here does not seem to be used by anything else from a github search - https://github.com/search?q=org%3AONSdigital+%2Fcollectionexercises%2Fsurveys&type=code

LJBabbage commented:
Just waiting for build to be fixed on this one
